skimage/feature/tests/test_corner.py

```diff
@@ -361,13 +361,33 @@ def test_corner_peaks():
                            threshold_rel=0)
     assert corners.shape == (2, 2)
 
-    corners = corner_peaks(response, exclude_border=False, min_distance=1)
+    corners = corner_peaks(response, exclude_border=False,
+                           min_distance=1, threshold_rel=0.1)
     assert corners.shape == (5, 2)
 
     corners = corner_peaks(response, exclude_border=False, min_distance=1,
-                           indices=False)
+                           indices=False, threshold_rel=0.1)
     assert np.sum(corners) == 5
 
+    # Test Euclidean distance
```

Comment:
Could this be separated into a new test? Tests should be as granular as possible so that just the name of a failed test gives as much information as possible about the failure.